website/default_config.py

```diff
@@ -10,6 +10,20 @@
     'v1': '1.0',
     'v2': '2.0',
 }
+
+ # Version dictionary format:  str_key: ( str_tag, ( int_major, int_minor, int_micro, str_type, int_development ), str_url )
+ # where:
+ #      str_key is the update path.  Valid keys are: 'stable', 'beta' and 'dev'.
+ #      str_tag is the version display tag (e.g.: '2.0.0beta3').
+ #      str_url is the dowload link for the version.
```

Comment:
I'd use a dictionnary for urls, it'd be more future-proof. Something like:

```
'urls': (
   'download': 'http://picard.musicbrainz.org/'
)
```

This could be easily extended if needed (ie. adding specific downloads urls, changelog urls, etc...)

Reply:
Just to make sure I understand before pushing a commit, are you looking for something like this?

    PICARD_VERSIONS = {
        'stable': (
            'tag': '2.0.1',
            'version': ( 2, 0, 1, 'final', 0),
            'urls': {
                'download': 'https://picard.musicbrainz.org/',
                'changelog': 'https://picard.musicbrainz.org/changelog/',
            }
        ),
        'beta': (
            'tag': '2.0.0.beta3',
            'version': ( 2, 0, 0, 'dev', 6),
            'urls': {
                'download': 'https://github.com/metabrainz/picard/releases/tag/2.0.0dev6',
                'changelog': 'https://github.com/metabrainz/picard/compare/2.0.0dev6...master',
            }
        ),
        'dev': (
            'tag': '2.0.2.dev1',
            'version': ( 2, 0, 2, 'dev', 1),
            'urls': {
                'download': 'https://github.com/metabrainz/picard',
            }
        ),
    }

Note that I changed each of the items to a dictionary for clarity.  is this going too far?

Reply:
> Just to make sure I understand before pushing a commit, are you looking for something like this?

Yes, exactly :+1:

Reply:
Thank-you.  Changes made and pushed to PR.